digital_image_processing/dithering/burkes.py

```diff
@@ -0,0 +1,98 @@
+"""
+Implementation Burke's algorithm (dithering)
+"""
+from cv2 import imread, imshow, waitKey, destroyAllWindows
+import numpy as np
+
+
+class Burkes:
+    """
+    Burke's algorithm is using for converting grayscale image to black and white version
+    Source: Source: https://en.wikipedia.org/wiki/Dither
+
+    Note:
+        * Best results are given with threshold= ~1/2 * max greyscale value.
+        * This implementation get RGB image and converts it to greyscale in runtime.
+    """
+
+    def __init__(self, input_img, threshold: int):
+        self.min_threshold = 0
+        self.max_threshold = int(
+            self.get_greyscale(255, 255, 255)
+        )  # max greyscale value for #FFFFFF
```

Comment:
```suggestion
        # max greyscale value for #FFFFFF
        self.max_threshold = int(self.get_greyscale(255, 255, 255))  
```

Reply:
How `black` should be used in this case? Calling ```black .``` make this lines wrapped like above.

Reply:
If the comment is not on the same line as the code then black will not wrap the code because the line is <= 88 characters.